visualization/app/codeCharta/ui/codeMap/threeViewer/threeViewerService.ts

```diff
@@ -1,30 +1,54 @@
 "use strict"
+
+import * as Stats from "stats-js"	// NOTE npm i stats-js
 import { ThreeSceneService } from "./threeSceneService"
 import { ThreeCameraService } from "./threeCameraService"
 import { ThreeOrbitControlsService } from "./threeOrbitControlsService"
 import { ThreeRendererService } from "./threeRendererService"
 import { ThreeUpdateCycleService } from "./threeUpdateCycleService"
+import { RenderPass } from 'three/examples/jsm/postprocessing/RenderPass.js'
+import { FXAAShader } from './shaders/FXAAShader.js'
+import { ShaderPass } from "three/examples/jsm/postprocessing/ShaderPass"
+import { WebGLInfo } from "three"
 
 export class ThreeViewerService {
+	private stats : Stats
+	private xPanel: any
+	private yPanel: any
+	private maxXPanel : number = 0
+	private maxYPanel : number = 0
+	private enableFXAA : boolean = false // TODO can be selective
```

Comment:
```suggestion
	private enableFXAA = false // TODO can be selective
```
The type can be inferred automatically because of the initialization with `false`.